<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyFilterGUIRow</class>
 <widget class="QWidget" name="CopyFilterGUIRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="modeComboBox"/>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="factorSpinBox">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="asteriskLabel">
     <property name="text">
      <string notr="true">*</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="channelComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
